mcux/hal_nxp.cmake

```diff
@@ -115,8 +115,9 @@ include_driver_ifdef(CONFIG_COUNTER_MCUX_PIT		pit		driver_pit)
 include_driver_ifdef(CONFIG_COUNTER_MCUX_RTC		rtc		driver_rtc)
 include_driver_ifdef(CONFIG_DAC_MCUX_DAC		dac		driver_dac)
 include_driver_ifdef(CONFIG_DAC_MCUX_DAC32		dac32		driver_dac32)
-include_driver_ifdef(CONFIG_DMA_MCUX_EDMA		edma		driver_edma)
 include_driver_ifdef(CONFIG_DMA_MCUX_EDMA		dmamux		driver_dmamux)
+include_driver_ifdef(CONFIG_DMA_MCUX_EDMA_V2		edma		driver_edma)
```

Comment:
Why don't we leave this as EDMA and call the new one EDMA_v3

Reply:
ok, i've updated